sockets/sockets.js

```diff
@@ -516,6 +535,12 @@ var actionsObj = {
             help: '/r: Reply to a mod who just messaged you.',
             run(data, senderSocket) {
                 const { args } = data;
+
+                // If the player has not been whispered to yet.
+                if (!lastWhisperObj[senderSocket.request.user.username]) {
```

Comment:
I just realised that this and my other code for lastWhisperObj may break if the user relogs back in with a different case username. E.g. ProNub -> pmmod/whisper/r -> relog in as PRONUB. Could you check this for me?